app/services/whatsapp/providers/whatsapp_cloud_service.rb

```diff
@@ -0,0 +1,110 @@
+class Whatsapp::Providers::WhatsappCloudService < Whatsapp::Providers::BaseService
+  def send_message(phone_number, message)
+    if message.attachments.present?
+      send_attachment_message(phone_number, message)
+    else
+      send_text_message(phone_number, message)
+    end
+  end
+
+  def send_template(phone_number, template_info)
+    response = HTTParty.post(
+      "#{phone_id_path}/messages",
+      headers: api_headers,
+      body: {
+        messaging_product: 'whatsapp',
+        to: phone_number,
+        template: template_body_parameters(template_info),
+        type: 'template'
+      }.to_json
+    )
+
+    process_response(response)
+  end
+
+  def sync_templates
+    response = HTTParty.get("#{business_account_path}/message_templates?access_token=#{whatsapp_channel.provider_config['api_key']}")
+    whatsapp_channel.update(message_templates: response['data'], message_templates_last_updated: Time.now.utc) if response.success?
+  end
+
+  def validate_provider_config
+    true
+  end
+
+  def api_headers
+    { 'Authorization' => "Bearer #{whatsapp_channel.provider_config['api_key']}", 'Content-Type' => 'application/json' }
+  end
+
+  def media_url(media_id)
+    "https://graph.facebook.com/v13.0/#{media_id}"
```

Comment:
Can we maintain the version at the top of the class? or use global_config['FACEBOOK_API_VERSION'] ? to use it for `phone_id_path`, `business_account_path` too, if it allows common versions.

Reply:
bussiness_account APIs are only available in v14.0, while our message payload implementation built for 360dialog is built over v13.0.  I left this as such because of that. 

Lets revisit this again and see if we can unify the version across Chatwoot.